model/file.go

```diff
@@ -22,6 +22,36 @@ type FileUploadResponse struct {
 	ClientIds []string    `json:"client_ids"`
 }
 
+const (
+	FILE_SORT_BY_CREATED      = "CreateAt"
+	FILE_SORT_BY_SIZE         = "Size"
+	FILE_SORT_BY_USERNAME     = "Username"
+	FILE_SORT_BY_CHANNEL_NAME = "Name"
+
+	FILE_SORT_ORDER_ASCENDING  = "ASC"
+	FILE_SORT_ORDER_DESCENDING = "DESC"
+)
+
+// GetFilesOptions contains options for getting files
+type GetFilesOptions struct {
```

Comment:
I know this name comes right out of the ticket, but I wonder if we should call this `GetFileInfoOptions` to match the terminology elsewhere.

Reply:
I wasn't sure of the name either. I decided to use `GetFilesOptions` instead of `GetFileInfoOptions` because I thought this could potentially be reused for other file query methods.

Reply:
I'll defer to whatever you think is appropriate.

Reply:
I'd be inclined to go with `FileInfo` as the underlying noun here, as it helps emphasize that this is the metadata and not the files themselves.